packages/plugins/__template/src/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview TODO: Add plugin overview.
+ */
+
+/**
+ * TODO: Rename plugin and add plugin description.
+ */
+export default class Plugin {
```

Comment:
I think using default is a bit controversial and we aren't using default for WorkspaceSearch.
Additionally, google javscript style guide says to not use default export.
https://google.github.io/styleguide/jsguide.html#es-module-exports